src/collection-browser.ts

```diff
@@ -493,12 +498,22 @@ export class CollectionBrowser
     `;
   }
 
+  private get facetTopTemplate(): TemplateResult {
+    return html`
+      <div id="facet-top-view" slot="facet-top">
+        load adjustable div height and width
+      </div>
+    `;
+  }
+
   /**
    * Template for the desktop version of the left column, displaying the facets sidebar.
    */
   private get desktopLeftColumnTemplate(): TemplateResult {
     return html`
       <div id="left-column" class="column">
+        ${this.isFacetTopView ? this.facetTopTemplate : nothing}
```

Comment:
we talked about just adding the slot here `<slot name='foo></slot>`
whereas in your demo, you toggle the component going into the slot 👍